uds/database/data_record/text_data_record.py

```diff
@@ -0,0 +1,73 @@
+"""Definition of TextDataRecord which is class for encode and decode values of data records."""
+
+__all__ = ["TextTableDataRecord"]
+
+from typing import Dict, Optional
+
+from .abstract_data_record import AbstractDataRecord, DataRecordPhysicalValueAlias, DecodedDataRecord
+from .raw_data_record import RawDataRecord
+
+
+class TextTableDataRecord(RawDataRecord, AbstractDataRecord):
```

Comment:
Let's rename it to `MappingDataRecord` as it is describing direcly what it does.